config/crd/bases/ndb.nutanix.com_databases.yaml

```diff
@@ -78,56 +73,73 @@ spec:
                             type: string
                           name:
                             type: string
+                        required:
+                        - id
+                        - name
                         type: object
                       dbParam:
                         properties:
                           id:
                             type: string
                           name:
                             type: string
+                        required:
+                        - id
```

Comment:
I am most likely reading this wrong ... but do we need both id and name? I thought any 1 was sufficient

Reply:
Turns out that, it was added because I removed "//+optional" from the struct definition. Adding it back remove these.